distribution/src/conf/internal-apis.xml

```diff
@@ -33,6 +33,26 @@
                             </user>
                         </users>
                     </UserStore>
+                </handler-->
+                <handler class="org.wso2.micro.integrator.management.apis.security.handler.JWTTokenSecurityHandler"
+                         name="JWTTokenSecurityHandler">
+                    <TokenStoreConfig>
+                        <MaxSize>200</MaxSize>
+                        <TokenCleanupTaskInterval>600000</TokenCleanupTaskInterval>
```

Comment:
Let's configure this in seconds. Also we need to add a comment, describing the unit

Reply:
Thanks. changed the config